src/routes/admin/wireless/create.ts

```diff
@@ -14,14 +14,14 @@ export async function createWirelessProfile (req: Request, res: Response): Promi
   const log = new Logger('createWirelessProfile')
   try {
     const passphrase = wirelessConfig.pskPassphrase
+    // prevents passphrase from being stored in DB
     if (req.secretsManager) {
       wirelessConfig.pskPassphrase = 'PSK_PASSPHRASE'
     }
-
     const results: WirelessConfig = await req.db.wirelessProfiles.insert(wirelessConfig)
     // store the password into Vault
     if (req.secretsManager) {
-      await req.secretsManager.writeSecretWithKey(`Wireless/${wirelessConfig.profileName}`, wirelessConfig.pskPassphrase, passphrase)
+      await req.secretsManager.writeSecretWithObject(`Wireless/${wirelessConfig.profileName}`, { PSK_PASSPHRASE: passphrase })
```

Comment:
We are just passing it straight in, but should the object being passed in be typed as WifiCredentials?